admin-manual/installation-setup/customization/scaling-archivematica.rst

```diff
@@ -0,0 +1,396 @@
+without defaults` will only generate thumbnail images for formats that have a 
+specific thumbnail rule defined. The default thumbnail rule only produces a 
+grey icon and for many formats has little value, but in transfers with many 
+files, can take significant processing time. 
+
+**Select compression algorithm:** AIP compression causes an AIP to be put into 
+a container (e.g. a 7Zip container).  Using containers makes AIP storage and 
+transfer easier because the AIP is easier to move around as a single file. The 
+AIP file size also has the potential to be reduced, which saves storage space 
+and speeds up transfers to external AIP stores. The disadvantage is that 
+compression can take significant processing time and resources. AIP compression 
+introduces three extra steps in the workflow: compression to create the 
+container, then decompression to allow for a final checksum validation step. In 
+transfers with very large numbers of files (thousands) we have seen significant
+performance improvements by not compressing the AIP.  
+
+**Select compression level:** Selecting a higher compression level means that 
+the resulting AIP is smaller, but compression also takes longer. Lower 
+compression levels mean quicker compression, but a larger AIP.
+
+Preservation Action Rules
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some of the default preservation action rules can take considerable processing 
+time and resources. We have found the following rules useful to change in some 
+cases. 
+
+**Turn off default characterization rule:** FITS is used to characterise files 
+that don't have a recognised file format. Executing this rule takes processing 
+time and adds raw output to the METS file that can be low value for some  
+formats. For example, in scientific datasets with large numbers of generic text 
+files, or binary files created by instruments in scientific experiments, the 
+output can be verbose without being useful. 
+
+**Reduce number of image characterization rules:** Archivematica has rules 
+defined for all image formats to use ExifTool, Mediainfo and ffprobe for 
+characterisation. Using multiple tools ensures as much characterization output 
+as possible, but also introduces some level of duplication. Only using one of 
+the three tools for certain formats may provide an adequate level of 
+characterization with the benefit of reducing processing time and the size of 
+the final AIP. 
+
+See :ref:`Characterization <characterization>` and 
+:ref:`Altering commands and rules <altering-commands-rules>` for more details.  
+
+General Configuration Settings  
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Checksum Algorithm:** In the :ref:`General settings <dashboard-general>` 
+screen you can select which checksum algorithm Archivematica will use during 
+the Assign UUIDs and checksums micro-service. For the purposes of fixity 
+checking, the MD5 algorithm is perfectly adequate and takes less processing 
```

Comment:
I wonder if we should be a little less prescriptive about the adequacy of MD5, not that I disagree with the statement. Maybe "the MD5 algorithm may be adequate for your use case and takes less... [etc]".